### ISSUE TITLE
Add pT spectra for energy scan

### ISSUE THREAD

**Issue by NGoetz** [smash-transport/smash-analysis#33]
Most available data for LHC energies are pT spectra. For better comparison it would be good to also include a pT spectra tab to the energy scan target.

**Comment by akschaefer** [comment]
This infrastructure is in principle already there. The analyzed pT spectra are already determined in the default energy scan run. I'm not sure if the plots are also created, but the required code to create them is also already there. It's probably just a matter of uncommenting a line or adding an observable to a list or similar. 
What happened in the past though (for example with some of the STAR data) is that we converted the pT spectra to mT spectra and then added the data points to the mT spectra plots. Because mT and pT spectra basically contain the same information.
If the idea is however to add an entirely new section to the energy scan target webpage, there's some more work to be done to create this additional subpage in the automatic html script.

**Comment by NGoetz** [comment]
Yes the idea is to add the subpage. But good to know that the pT spectra are already determined!
